classes/app.rb

```diff
@@ -4,7 +4,8 @@
 require_relative 'student'
 require_relative 'teacher'
 require_relative 'classroom'
-
+require_relative 'preserve_data'
+# rubocop:disable Metrics/ClassLength
```

Comment:
For the reviewer:

We have added the lines

```
# rubocop:disable Metrics/CyclomaticComplexity
# rubocop:enable Metrics/CyclomaticComplexity 
```

Because of the these errors

![Screenshot from 2023-11-13 18-04-53](https://github.com/siddghosh108/school-library/assets/48357105/38121f8f-b087-431f-a297-260bf8ba538d)


![Screenshot from 2023-11-13 18-05-05](https://github.com/siddghosh108/school-library/assets/48357105/fc07615c-581d-4538-993a-166d5755352d)


As you can see, the 7 `when` cases are not that much and if we wanted to implement a work-around we will make the program unnecessarily more complex, so it makes sense to disable this "error"

We are not disabling linters, we are just turning off a specific warning.

Thank you.